### PR TITLE
[HTML markup] Footer markup

### PR DESCRIPTION
**Pull request by rudenko-vl**
- Add footer markup
- Add footer classes to markup

### CODE REVIEW

File: src/partials/layout/_footer.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<footer class="footer">
+  <div class="container">
+    <div class="footer-wrapper">
```
Comment (Androborn) [inline src/partials/layout/_footer.html:3]: You have container div already, why do you need wrapper div in it?

File: src/partials/layout/_footer.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      <p class="footer-wrapper__text">© 2020 | All Rights Reserved |</p>
```
Comment (Androborn) [inline src/partials/layout/_footer.html]: Rename class to footer-wrapper__copyright.
If you need to apply the same text styles to it, as to the next element - use mixed class.

File: src/partials/layout/_footer.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      <p class="footer-wrapper__text">
```
Comment (Androborn) [inline src/partials/layout/_footer.html]: This whole paragraph must have class __about-developers, not just __text.
If you need to apply the same text styles to it, as to the next element - use mixed class.

File: src/partials/layout/_footer.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+        Developed with<svg class="footer-wrapper__icon" width="14" height="13">
+          <use href="./images/sprite.svg#icon-heart"></use>
+        </svg>
+        <span>by</span>
+        <span class="footer-wrapper__about-developers">GoIT Students</span>
```
Comment (Androborn) [inline src/partials/layout/_footer.html]: Add mix classes "link footer-wrapper__link" here and add necessary styling to footer-wrapper__link, remove footer-wrapper__about-developers class.

File: src/partials/layout/_footer.html (hunk 1 above)
Comment (rudenko-vl) [reply src/partials/layout/_footer.html:3]: I removed the div container. Not needed for positioning.

File: src/partials/layout/_footer.html (hunk 2 above)
Comment (rudenko-vl) [reply src/partials/layout/_footer.html]: Renamed footer-wrapper__text to footer-wrapper__copyright.

File: src/partials/layout/_footer.html (hunk 3 above)
Comment (rudenko-vl) [reply src/partials/layout/_footer.html]: Сhanged class to __about-developers

File: src/partials/layout/_footer.html (hunk 4 above)
Comment (rudenko-vl) [reply src/partials/layout/_footer.html]: Changed to footer-wrapper__link

**Review by Androborn — APPROVED** [review]

**Pull request merged**